djangocms_alias/filters.py

```diff
@@ -82,6 +85,37 @@ def choices(self, changelist):
             }
 
 
+class CategoryFilter(admin.SimpleListFilter):
+    title = _("Category")
+    parameter_name = CATEGORY_FILTER_PARAM
+
+    def lookups(self, request, model_admin):
+        qs = model_admin.get_queryset(request)
+        cat_id = qs.values_list('alias__category', flat=True).distinct()
+        cat = Category.objects.filter(pk__in=cat_id).order_by('translations__name')
+        for obj in cat:
+            yield str(obj.pk), smart_text(obj)
```

Comment:
smart_text is deprecated in django 3: https://docs.djangoproject.com/en/4.0/releases/3.0/#features-deprecated-in-3-0

some help to replace: https://docs.djangoproject.com/en/3.2/_modules/django/utils/encoding/#smart_text